src/com/aca/homework/week3/stack/strings/Stack.java

```diff
@@ -0,0 +1,14 @@
+package com.aca.homework.week3.stack.strings;
+
+public class Stack {
+    private String[] stack = new String[10];
+    private int numOfElements = 0;
+    public void push(String s){
+        stack[numOfElements++] = s;
+    }
+
+    public void pop(){
```

Comment:
let's change this to return a String so you can print it outside of this class

Reply:
have just applied the changes